### ISSUE TITLE
Add sponsors section to bottom of website

### ISSUE THREAD

**Issue by errcHuang** [ronnyb29/medhacksSite#54]
See http://www.calhacks.io/, https://www.treehacks.com/ for inspirations. We should have two sections: "Sponsors" and "Partners"

Under partners we should put MLH (logos here https://mlh.io/brand-guidelines)

**Comment by ronboger** [comment]
yep, we also have an example from last year :)

**Comment by errcHuang** [comment]
Add TechVentures

**Comment by ronboger** [comment]
logo is also there from last year in the assets folder

**Comment by errcHuang** [comment]
Also to establish credibility, we should have a section titled "Last Year's Sponsors" and list the ones from last year since we did have quite a few partners/sponsors last year. 

https://github.com/ronnyb29/medhacksSite/blob/499a8b6157e263d04e672d26a219a27ca89a4392/index.html (list of last year's is in there).

**Comment by ronboger** [comment]
agreed, until we get a critical mass of new sponsors from this year

i also think it's important to add back a lot of the FAQ from last year

**Comment by vicmart** [comment]
which FAQ specifically?
